### PR TITLE
add results to google sheets and index results into ES

### PR DESCRIPTION
**Pull request by mohit-sheth**
..and It will upload the csv to the pbench server.
some tab/spaces corrections
Results: [google sheet](https://docs.google.com/spreadsheets/d/1z-F5ROeWk0c4ShcVzYsgHtOVVaP036VsofZaXoemfdM/edit#gid=1071183342) and pbench [results.csv](http://pbench.perf.lab.eng.bos.redhat.com/results/EC2::scale-ci-http-6f989/http-2020-04-21_18.35.43-mix/pbench-user-benchmark_0001r-001cpt-0000d_ms-001ka-ytlsru-10s-http-mix_2020.04.21T18.34.19/)

### CODE REVIEW

File: workloads/vars/http.yml [hunk 1]
```diff
@@ -28,6 +28,10 @@ job_completion_poll_attempts: "{{ lookup('env', 'JOB_COMPLETION_POLL_ATTEMPTS')|
 
 # HTTP workload specific parameters
 http_env_vars: [
+  "ES_SERVER",
```
Comment (chaitanyaenr) [inline workloads/vars/http.yml:31]: The elastic search vars needs to be added to the docs.

**Comment by chaitanyaenr** [comment]
Fixes https://github.com/openshift-scale/workloads/issues/132.

File: workloads/vars/http.yml (hunk 1 above)
Comment (mohit-sheth) [reply workloads/vars/http.yml:31]: Done

**Comment by mohit-sheth** [comment]
results [with](https://docs.google.com/spreadsheets/d/1rD4gcXm1Qel32jVDBFf4MkXAwgak-fZh2yBqSroLKqk/edit#gid=609548994) and [without](https://docs.google.com/spreadsheets/d/1_e_DDcL4W1l_naMqY0ANMfJL5obYS8EtskLvtRul_rM/edit#gid=358781662) COMPARE_RESULTS
@chaitanyaenr

**Review by dry923 — APPROVED** [review]
LGTM

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**